tutor/commands/compose.py

```diff
@@ -365,6 +370,80 @@ def copyfrom(
     )
 
 
+@click.command(
+    help="TODO describe"
+)
+@click.argument(
+    "mount_paths",
+    metavar="mount_path",
+    nargs=-1,
+    type=click.Path(dir_okay=True, file_okay=False, resolve_path=True),
+)
+@click.pass_obj
+def copyartifacts(context: BaseComposeContext, mount_paths: list[Path]) -> None:
```

Comment:
I'm wondering if it makes sense to provide a dedicated command. In which cases will users have to run it? If I understand correctly, they will only run it when they don't want to run `launch`. But then this opens up the possibility that they will forget to build the Docker image before running `copyartifacts`. In such a scenario, the artifacts will be outdated, right?

I might not 100% understand the pros and cons, but it seems to me that it would be more intuitive if artifacts are copied as part of `do init`.